pkg/runner/runner.go

```diff
@@ -220,16 +220,20 @@ func (r *runner) monitor(ctx context.Context, organizationId string, environment
 			lastResult = execution.Result
 		}
 		if !lastResult.IsFinished() {
+			workflowStatus := testkube.ABORTED_TestWorkflowStatus
+			if lastResult.Status == common.Ptr(testkube.ABORTED_TestWorkflowStatus) {
+				workflowStatus = testkube.ABORTED_TestWorkflowStatus
```

Comment:
it'd already workflowStatus := testkube.ABORTED_TestWorkflowStatus

Reply:
fixed in latest push